ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -1826,3 +1827,65 @@ def test_catalog_list_upgradable_packages(
     assert pkg.version == '2015-09'
     assert pkg.size == '200KB'
     assert isinstance(pkg, ZippedZim)
+
+
+def test_catalog_list_nothandled_packages(
+        tmpdir, settings, testdatadir, mocker):
+    from ideascube.serveradmin.catalog import Catalog
+
+    zippedzim = testdatadir.join('catalog', 'wikipedia.tum-2015-08')
+    path = tmpdir.mkdir('source').join('wikipedia_tum_all_nopic_2015-08.zim')
+    zippedzim.copy(path)
+
+    remote_catalog_file = tmpdir.join('source').join('catalog.yml')
+    with remote_catalog_file.open(mode='w') as f:
+        f.write('all:\n')
```

Comment:
I also remember specifically not wanting to use `yaml.safe_dumps` here, because... Well I don't remember why.

Maybe I thought that since the API being tested uses `yaml`, it was better to have the tests use something else? (to avoid making the same mistake in the tests as in the API, which would then cover up that mistake, getting the tests to pass)

In hindsight, that would have been a poor reason in this specific case.

But I can't remember if that was it, or if I had a really good reason. :disappointed: